[2023-04-06 01:23:13,156] INFO _log() L224   * Restarting with stat - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-06 01:23:15,539] INFO _log() L224  127.0.0.1 - - [06/Apr/2023 01:23:15] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-06 01:23:15,636] INFO _log() L224  127.0.0.1 - - [06/Apr/2023 01:23:15] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-06 01:23:15,639] INFO _log() L224  127.0.0.1 - - [06/Apr/2023 01:23:15] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-06 01:23:15,643] INFO _log() L224  127.0.0.1 - - [06/Apr/2023 01:23:15] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-06 01:23:15,714] INFO _log() L224  127.0.0.1 - - [06/Apr/2023 01:23:15] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-06 01:23:15,773] INFO _log() L224  127.0.0.1 - - [06/Apr/2023 01:23:15] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-06 01:23:21,284] INFO _log() L224  127.0.0.1 - - [06/Apr/2023 01:23:21] "GET /clientes HTTP/1.1" 200 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-06 01:23:27,937] ERROR log_exception() L1449 Exception on /cliente [DELETE] - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\flask\app.py L1449
Traceback (most recent call last):
  File "C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\flask\app.py", line 1520, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\flask\app.py", line 1539, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\flask\app.py", line 1695, in make_response
    raise TypeError(
TypeError: The view function for 'del_cliente' did not return a valid response. The function either returned None or ended without a return statement.
[2023-04-06 01:23:27,939] INFO _log() L224  127.0.0.1 - - [06/Apr/2023 01:23:27] "[35m[1mDELETE /cliente HTTP/1.1[0m" 500 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-06 01:24:24,349] INFO _log() L224   * Detected change in 'C:\\PUC-RIO\\Sprint-1\\BackEnd\\puc-rio-mvp-sprint-1-back-end\\app.py', reloading - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-06 01:24:24,431] INFO _log() L224   * Restarting with stat - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-06 01:24:30,820] WARNING del_cliente() L128  Erro ao excluir o cliente #'11', Cliente não encontrado na base :/ - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\app.py L128 
[2023-04-06 01:24:30,821] INFO _log() L224  127.0.0.1 - - [06/Apr/2023 01:24:30] "[33mDELETE /cliente HTTP/1.1[0m" 404 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-06 01:24:37,182] ERROR log_exception() L1449 Exception on /cliente [DELETE] - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\flask\app.py L1449
Traceback (most recent call last):
  File "C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\flask\app.py", line 1520, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\flask\app.py", line 1539, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\flask\app.py", line 1695, in make_response
    raise TypeError(
TypeError: The view function for 'del_cliente' did not return a valid response. The function either returned None or ended without a return statement.
[2023-04-06 01:24:37,185] INFO _log() L224  127.0.0.1 - - [06/Apr/2023 01:24:37] "[35m[1mDELETE /cliente HTTP/1.1[0m" 500 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-06 01:25:24,095] INFO _log() L224   * Detected change in 'C:\\PUC-RIO\\Sprint-1\\BackEnd\\puc-rio-mvp-sprint-1-back-end\\app.py', reloading - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-06 01:25:24,187] INFO _log() L224   * Restarting with stat - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-06 01:26:51,217] INFO _log() L224   * Detected change in 'C:\\PUC-RIO\\Sprint-1\\BackEnd\\puc-rio-mvp-sprint-1-back-end\\app.py', reloading - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-06 01:26:51,298] INFO _log() L224   * Restarting with stat - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-06 01:27:53,466] INFO _log() L224   * Detected change in 'C:\\PUC-RIO\\Sprint-1\\BackEnd\\puc-rio-mvp-sprint-1-back-end\\app.py', reloading - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-06 01:27:53,546] INFO _log() L224   * Restarting with stat - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-06 01:27:56,352] INFO _log() L224  127.0.0.1 - - [06/Apr/2023 01:27:56] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-06 01:27:56,433] INFO _log() L224  127.0.0.1 - - [06/Apr/2023 01:27:56] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-06 01:27:56,442] INFO _log() L224  127.0.0.1 - - [06/Apr/2023 01:27:56] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-06 01:27:56,448] INFO _log() L224  127.0.0.1 - - [06/Apr/2023 01:27:56] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-06 01:27:56,507] INFO _log() L224  127.0.0.1 - - [06/Apr/2023 01:27:56] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-06 01:27:56,564] INFO _log() L224  127.0.0.1 - - [06/Apr/2023 01:27:56] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-06 01:28:05,230] INFO _log() L224  127.0.0.1 - - [06/Apr/2023 01:28:05] "GET /clientes HTTP/1.1" 200 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-06 01:28:11,175] INFO _log() L224  127.0.0.1 - - [06/Apr/2023 01:28:11] "[35m[1mDELETE /cliente HTTP/1.1[0m" 204 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-06 01:29:12,479] INFO _log() L224   * Detected change in 'C:\\PUC-RIO\\Sprint-1\\BackEnd\\puc-rio-mvp-sprint-1-back-end\\app.py', reloading - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-06 01:29:12,540] INFO _log() L224   * Restarting with stat - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-06 01:29:23,843] INFO _log() L224  127.0.0.1 - - [06/Apr/2023 01:29:23] "[35m[1mDELETE /cliente HTTP/1.1[0m" 500 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
